docker/docker-compose.ubuntu-1604.403.yaml

```diff
@@ -1,14 +0,0 @@
-version: "3"
-
-services:
-
-  runtime-setup:
-    image: swift-nio-http2:ubuntu-16.04-4.0.3
-    build:
-      args:
-        ubuntu_version : "16.04"
-        swift_version : "4.0.3"
-        install_nghttp2_from_source: "true"
```

Comment:
why are we removing the whole docker yamls here?

Reply:
ah, we don't support 4.0 :)

Reply:
Because we don't support Swift 4.0, and never have. This is mentioned in the commit message.